lib/trento/infrastructure/installation/installation.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Trento.Installation do
+defmodule Trento.Infrastructure.Installation do
```

Comment:
Maybe this should even be a `context` itself, what do you think?

Reply:
I was thinking about that, the same mixed feeling with alerting, I don't know if it "qualifies" as a phoenix context, but yes I have the same feeling, let's investigate further